watertap/core/tests/test_wt_database.py

```diff
@@ -14,48 +14,28 @@
 Tests for WaterTAP database wrapper
 """
 import pytest
-import os
+from pathlib import Path
 
 from watertap.core.wt_database import Database
 
 
 @pytest.mark.unit
 def test_default_path():
     db = Database()
-
-    assert os.path.normpath(db._dbpath) == os.path.normpath(
-        os.path.join(
-            os.path.dirname(os.path.abspath(__file__)),
-            "..",
-            "..",
-            "data",
-            "techno_economic",
-        )
-    )
+    assert db._dbpath == Path(__file__).parent.parent.parent / "data/techno_economic"
 
 
 @pytest.mark.unit
 def test_invalid_path():
-    with pytest.raises(
-        OSError,
-        match="Could not find requested path foo. Please "
-        "check that this path exists.",
```

Comment:
We generally like having these match statements so we can be sure the "right" error was raised. But perhaps not strictly needed.

Reply:
Pushed up a commit to address this.